Firmware/RTK_Surveyor/FileSdFatMMC.h

```diff
@@ -195,21 +214,21 @@ class FileSdFatMMC : public SdFile
 #endif
 };
 
-//Update the file access and write time with date and time obtained from GNSS
-// These are SdFile-specific. SD_MMC does this automatically
+// Update the file access and write time with date and time obtained from GNSS
+//  These are SdFile-specific. SD_MMC does this automatically
```

Comment:
Double space! *gasp*

Reply:
Humm. That _might_ be a VSC whitespace thing? Personally I like the space. Happy to be pushed off the fence either way...